.github/workflows/test.yml

```diff
@@ -5,10 +5,11 @@ concurrency:
   cancel-in-progress: true
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-20.04, macos-latest]
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
         include:
           - python-version: "3.10"
```

Comment:
Try adding `os: ubuntu-20.04` to this block. I think that should sort out the coverage issue, by just not measuring coverage on MacOS.

Reply:
Done.